core/storage/opportunity/gae_models_test.py

```diff
@@ -121,6 +121,38 @@ def test_get_deletion_policy(self) -> None:
             .get_deletion_policy(),
             base_models.DELETION_POLICY.NOT_APPLICABLE)
 
+    def test_get_model_association_to_user(self) -> None:
+        self.assertEqual(
+            opportunity_models.ExplorationOpportunitySummaryModel
+            .get_model_association_to_user(),
```

Comment:
Can you indent this line so that it's clear that this is all one statement? Ditto elsewhere